mayastor/src/bdev/nvmx/controller.rs

```diff
@@ -794,11 +794,11 @@ pub extern "C" fn nvme_poll_adminq(ctx: *mut c_void) -> i32 {
     let mut context = NonNull::<TimeoutConfig>::new(ctx.cast())
         .expect("ctx pointer may never be null");
     let context = unsafe { context.as_mut() };
-
     let rc =
         unsafe { spdk_nvme_ctrlr_process_admin_completions(context.ctrlr) };
     if rc < 0 {
-        context.reset_controller();
+        //  unsafe { spdk_nvme_ctrlr_fail(context.ctrlr) };
+        // context.reset_controller();
```

Comment:
Is it safe to ignore admin command completion errors? Is this related to KATO being disabled (in SPDK, at least)?

Reply:
No, it is not. We don't want to reset because it will lock the thread up with the connection to something that is not there. Eventually, we might want to revisit this and say that we want to wait for a few seconds or retry a couple of times etc.